ts/features/itwallet/common/components/ItwIssuanceMetadata.tsx

```diff
@@ -104,6 +105,18 @@ export const ItwIssuanceMetadata = ({
     ? getItwAuthSource(credential)
     : getAuthSource(credential);
 
+  const releasedByLabel = useMemo(
+    () =>
+      I18n.t(
+        `features.itWallet.verifiableCredentials.claims.${
+          itwCredential && credential.credentialType === CredentialType.PID
+            ? "releasedByPid"
+            : "releasedBy"
+        }`
+      ),
+    [credential.credentialType, itwCredential]
+  );
```

Comment:
Can we move the key directly to the component? I think `useMemo` could be avoided here.